tests/e2e/interface.spec.js

```diff
@@ -0,0 +1,96 @@
+module.exports = {
+  "Validators search": async function(browser) {
+    // move to according page
+    browser.url(browser.launch_url + "/validators", async () => {
+      // wait until validators list load
+      await browser
+        .execute(function() {
+          return new Promise(resolve => {
+            let attempts = 5
+            const f = () => {
```

Comment:
no abbreviations

Reply:
You mean f()? But this is a common name for a function that will be executed only in this block. Any suggestions about the name?

Reply:
see my refactoring removing the need for this. We have a code guideline to never use abbreviations. Always describe what this does. If you can't describe what this does, this probably needs to be refactored.